python/rmm/allocators/torch.py

```diff
@@ -16,11 +16,14 @@
 except ImportError:
     rmm_torch_allocator = None
 else:
-    import rmm._lib.torch_allocator
+    import pathlib
 
-    _alloc_free_lib_path = rmm._lib.torch_allocator.__file__
     rmm_torch_allocator = CUDAPluggableAllocator(
-        _alloc_free_lib_path,
+        pathlib.Path(__file__).absolute().parent
+        / ".."
+        / "_lib"
+        / "libtorch_allocator.so",
         alloc_fn_name="allocate",
         free_fn_name="deallocate",
     )
+    del pathlib
```

Comment:
Maybe cleaner to do the entire path computation in the else block with the import then delete immediately? That way the two are more obviously coupled.